scripts/network.js

```diff
@@ -20,14 +21,96 @@ if (networkEnabled) {
       domBalanceReloadStaking.classList.remove("playAnim");
       if (data.backend.blocks > cachedBlockCount) {
         console.log("New block detected! " + cachedBlockCount + " --> " + data.backend.blocks);
-        getUTXOsHeavy();
+        getUTXOs();
         getStakingRewards();
       }
       cachedBlockCount = data.backend.blocks;
     }
     request.send();
   }
 
+
+  /**
+   * Parses UTXOs and puts them in the mempool
+   * @param {Array<Object>} arrUTXOs - Array of object-formatted UTXOs
+   * @returns {Promise<void>} Resolves when it has parsed every UTXO
+   */
+  async function acceptUTXO(arrUTXOs) {
+    const nTimeSyncStart = Date.now() / 1000;
+    
+    for (const cUTXO of arrUTXOs) {
+      const cTx = await (await fetch(`${cExplorer.url}/api/v2/tx-specific/${cUTXO.txid}`)).json()
```

Comment:
as we already discussed on discord this code can be even more optimized
cUTXO already contains many info about the tx... hence you can already check if it is in the mempool in CONFIRMED state.
In this case fetching the specific data of the cTx is a useless request that can be removed

Reply:
If we can remove this without side effects (I think I did this to support Cold Staking after the OP_CODE bug caused explorer API issues), then this would be **awesome**, also makes the sync instant, since it's a single request.

Reply:
Changed the code a bit to implement this more cleanly